### PR TITLE
Accounts hash calculation metrics

### PR DESCRIPTION
**Pull request by sakridge**
#### Problem

Not sure what is taking time in bank_forks:set_root

#### Summary of Changes

Add timing for hash calculation.

Fixes #

### CODE REVIEW

**Comment by carllin** [comment]
hehe beat me and @t-nelson to it, thanks!

File: runtime/src/accounts_db.rs [hunk 1]
```diff
@@ -1592,9 +1592,16 @@ impl AccountsDB {
         }
 
         scan.stop();
-        debug!("{}", scan);
 
-        Ok(Self::accumulate_account_hashes(hashes))
+        let mut accumulate = Measure::start("accumulate");
```
Comment (carllin) [inline runtime/src/accounts_db.rs:1596]: Should the scan above also be included in the measurement?

File: runtime/src/accounts_db.rs [hunk 2]
```diff
@@ -1592,9 +1592,16 @@ impl AccountsDB {
...
+        let accumulated_hash = Self::accumulate_account_hashes(hashes);
+        accumulate.stop();
+        datapoint_info!(
+            "update_accounts_hash",
+            ("accounts_scan", scan.as_us(), i64),
+            ("hash_accumulate", accumulate.as_us(), i64),
```
Comment (carllin) [inline runtime/src/accounts_db.rs:1602]: Do we also want to add a metric for number of accounts/hashes acumulated?

Comment (sakridge) [reply runtime/src/accounts_db.rs:1603]: added

**Comment by t-nelson** [comment]
I picked this on top of v1.2.19 and it's running on the devnet validator, 47Sbuv6jL7CViK9F2NMW51aQGhfdpUu7WNvKyH645Rfi

**Pull request merged**